src/css/main.scss

```diff
@@ -1,4 +1,8 @@
 ---
+# These lines are for Jekyll
```

Comment:
The 6 dashes are a Jekyll thing, so Jekyll knows it's has to do something with the file.